### ISSUE TITLE
Remember the Milk not generating a YOUR_FROB

### ISSUE THREAD

**Issue by smacdonald23** [kevintuhumury/dashing-milkman#1]

**Comment by brendantay** [comment]
can anyone fix this? would really love to use this!

**Comment by kevintuhumury** [comment]
@smacdonald23 For some reason I haven't seen this issue (or forgot about it, I honestly don't remember). Sorry, about that. 

I haven't been using this gem or RTM at all anymore, but I'll try to take a look at this issue this week. I don't have a lot of free time, but I'll try to fix it. They might have changed the API or something like that. /cc @brendantay

**Comment by brendantay** [comment]
@kevintuhumury Thank you for the reply :) no rush if you don't have the time, would just be handy if it was able to work eventually if possible haha :)

Just to note, the error is with the generated to get the FROB, when navigating to the URL Provided when running 

`milkman authorize API_KEY SHARED_SECRET`

You get a link and you navigate to it and see the expected `API Application wants to link to your Remember The Milk account
You should not authorize API Application unless you trust them with access to your account.`

When you click on "Okay, ill allow it" you receive the following,

`Uh oh... something went wrong. Dont worry though it's not your fault. The application youre using to interact with Remember The Milk made an error. It might be worthwhile contacting the author of the application and telling them you encountered this page.

Alternatively you might like to wait a little while and then try again`

**Comment by skyzo35** [comment]
@brendantay do you have this issue when you copy/past the link ?
i have this issue too... maybe i'm not doing it well ?
maybe i have to be logged to the RTM account ?

**Comment by kevintuhumury** [comment]
@smacdonald23 I'm sorry this took so long to get fixed, but like I said before I don't have a lot of time to maintain the `milkman` gem and I don't use it anymore either.

I've gotten around to fix it though. I've updated the `milkman` gem from `v0.0.3` to `v0.0.4` which fixes this issue. Besides that, I've also released `v0.0.5` which uses the RTM API version 2 by default.

/cc @brendantay and @skyzo35
